app/brave_generated_resources.grd

```diff
@@ -265,12 +265,12 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
 
       <!-- Extensions -->
       <message name="IDS_UNVETTED_EXTENSION_INSTALL_PROMPT_TITLE" desc="Titlebar of the extension or app installation prompt which was not vetted by Brave. Asks the user if they want to install a particular extension or app.">
-        Brave does not review extensions for security and safety. Install this extension only if you trust the developer.
+        Brave does not review extensions for security and safety. Install this extension only if you trust the developer
```

Comment:
This text is not part of the settings page. It should have the period at the end.